charts/ocis/templates/frontend/deployment.yaml

```diff
@@ -1,4 +1,8 @@
-{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameFrontend" "appNameSuffix" "") -}}
+{{ if .Values.features.appsIntegration.enabled }}
+{{- range $officeSuite := .Values.features.appsIntegration.wopiIntegration.officeSuites }}
+{{ if $officeSuite.enabled }}
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameFrontend" "appNameSuffix" (regexReplaceAll "\\W+" (lower $officeSuite.name) "-")) -}}
+---
```

Comment:
I think there is only one frontend service.

Reply:
could u please help me with it?

Reply:
Changed logic now to search all office suites first and select the correct one. If several offices have secureView enabled helm will fail with a message. -> https://github.com/owncloud/ocis-charts/pull/690/commits/83765921d31128ee39a79b54e1cb03dbb83fb8b8